src/styles/blocks/stars.css

```diff
@@ -0,0 +1,23 @@
+.stars {
+  margin: 0;
+
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
```

Comment:
Don't forget to remove margin-right for the last star in the row

Reply:
can i use gap?